src/main/java/edu/hawaii/its/api/service/GroupingAssignmentService.java

```diff
@@ -41,29 +37,35 @@ public class GroupingAssignmentService {
     public static final Log logger = LogFactory.getLog(GroupingAssignmentService.class);
 
     @Autowired
-    private GrouperApiService grouperApiService;
+    private MemberService memberService;
 
     @Autowired
-    private MemberService memberService;
+    private GrouperApiService grouperApiService;
 
     @Autowired
     private GroupingsService groupingsService;
 
+    /**
+     * A list of grouping paths for all groupings, restricted to admins' use only.
+     */
+    public GroupingPaths groupingPaths(String adminUhIdentifier) {
```

Comment:
Sorry for the confusion with the naming, this function should definitely be called allGroupingPaths.